inc/cleantalk-admin.php

```diff
@@ -1216,3 +1216,49 @@ function apbct__add_debug_tab($links)
 
     return $links;
 }
+
+/**
+ * Function for `wpmu_blogs_columns` filter-hook.
+ *
+ * @param string[] $sites_columns An array of displayed site columns.
+ *
+ * @return string[]
+ */
+function apbct__wpmu_blogs_columns_filter($sites_columns)
+{
+    $sites_columns[] = esc_html__('CleanTalk Status', 'cleantalk-spam-protect');
+
+    return $sites_columns;
+}
+
+add_filter('wpmu_blogs_columns', 'apbct__wpmu_blogs_columns_filter');
+
+/**
+ * Function for `manage_posts_custom_column` action-hook.
+ *
+ * @param string $_column_name The name of the column to display.
+ * @param int    $site_id     The current post ID.
+ *
+ * @return void
+ */
+function apbct__manage_sites_custom_column_action( $_column_name, $site_id )
+{
+    $cleantalk_data = get_blog_option($site_id, 'cleantalk_data');
+    $key_is_ok_text = esc_html__('The Access key is filled out and correct', 'cleantalk-spam-protect');
+    $key_is_bad_text = esc_html__('The Access key has not been filled or incorrect.', 'cleantalk-spam-protect');
```

Comment:
Please, use these phrases and no uppercase in the "access key". "The access key is set and correct."/"The access key is not set or is incorrect."

Reply:
DONE